### PR TITLE
Fix CMakeLists bug on Sofa.ini creation

### PR DESCRIPTION
**Pull request by epernod**
Small fix related to issue #114 .

Have tested the compilation without the flag `SOFA_INSTALL_RESOURCES_FILES` (which is set by default), Sofa.ini which is necessary is well created.

______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by guparan** [comment]
Hahaha I didn't see you already opened a PR for this!

File: CMakeLists.txt [hunk 1]
```diff
@@ -204,12 +211,9 @@ if(SOFA_INSTALL_RESOURCES_FILES)
...
-    # directory, it contains to relative paths to the installed resource directory.
-    set(SHARE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/share")
-    set(EXAMPLES_DIR "${CMAKE_CURRENT_SOURCE_DIR}/examples")
-    configure_file(${SOFA_KERNEL_SOURCE_DIR}/etc/sofa.ini.in "${CMAKE_BINARY_DIR}/etc/sofa.ini")
+	
+	# Create etc/installedSofa.ini: it contains the paths to share/ and examples/. Compare to the Sofa.ini in the
+	# build directory, this one contains relative paths to the installed resource directory.
     set(SHARE_DIR "../share/sofa")
     set(EXAMPLES_DIR "../share/sofa/examples")
     configure_file(${SOFA_KERNEL_SOURCE_DIR}/etc/sofa.ini.in "${CMAKE_BINARY_DIR}/etc/installedSofa.ini")
```
Comment (guparan) [inline CMakeLists.txt]: Why didn't you also remove this part from the `if(SOFA_INSTALL_RESOURCES_FILES)` block?
If we install with `SOFA_INSTALL_RESOURCES_FILES = false`, _src/etc/installedSofa.ini_ won't be created and _build/install/etc/sofa.ini_ neither.

Comment (epernod) [reply CMakeLists.txt]: I did the change quickly. I thought it was part of the install resources files.
I have committed the change. And also into the sofaKernel/CMakeLists.txt 
Btw why is there another install command in this file? Is it to be able to build/install sofaKernel lib without using the main Sofa CMakeLists.txt ?

**Comment by guparan** [comment]
Could you recreate this PR from a feature branch (not your master) and rebase it to remove the irrelevant merge commits ? I can do the rebase part if necessary ;-)

**Comment by epernod** [comment]
Duplicated with #291 which has been cleaned from unwanted commits

**Pull request closed without merging**